docs/modules/extend/examples/pdf-converter-article-title-with-author-and-date.rb

```diff
@@ -0,0 +1,44 @@
+class PDFConverterArticleTitleWithAuthorAndDate < (Asciidoctor::Converter.for 'pdf')
+  register_for 'pdf'
+
+  def ink_general_heading doc, title, opts
+    return super unless opts[:role] == :doctitle # <1>
+    ink_document_title title, opts # <2>
+    ink_document_metainfo doc, opts # <3>
+    ink_titlesection_body_separator # <4>
```

Comment:
I don't think the ink_titlesection_body_separator method is necessary here. I would prefer this logic to be moved back into this method.